README.md

```diff
@@ -104,11 +105,47 @@ And automatically fix issues:
 npm run lint:fix
 ```
 
+## Contribute
+### Pullrequests
+- Create a remote branch that fits the following naming convention: `pr/{package}-{description-of-contribution}`
+- Push commits to this branch. Commits should be small and the changes related. Set a commit message as described bellow.
```

Comment:
"Commits should be small and the changes related" sounds weird to me. Maybe "Commits and the related changes should be small"?